### DISCUSSION TITLE
Hardening script update v2023.6.29

### DISCUSSION THREAD

**Discussion by HotCakeX** [HotCakeX/Harden-Windows-Security#55] [General]
# First release of this repository 🥳

This is the first time I'm publishing a release and planning to do this for every new version of the hardening script or WDACConfig PowerShell module in the future.

It will send notifications to the users who are watching this repository letting them know there is a new version available.
It also allows me to offer proper change logs for each change.

The entire change log history of the hardening script is [available in Excel online](https://1drv.ms/x/s!AtCaUNAJbbvIhuVQhdMu_Hts7YZ_lA?e=df6H6P)

## Change log:

1.  Added Exploit Protection/Process Mitigations for [various apps](https://github.com/HotCakeX/Harden-Windows-Security/blob/main/Payload/ProcessMitigations.csv) such as Microsoft Edge (All channels), Quick Assist and some system processes. **More apps and processes will be added to the list once they are properly validated and confirmed to be fully compatible.**
    - Thanks to everyone participated in [this issue](https://github.com/HotCakeX/Harden-Windows-Security/issues/49)
3. Added back the `PrimaryPasswordSetting` policy to Edge after [confirming the bug related to it was fixed](https://techcommunity.microsoft.com/t5/microsoft-security-baselines/security-baseline-for-microsoft-edge-version-114/bc-p/3859142#M658).

<hr /><em>This discussion was created from the release <a href='https://github.com/HotCakeX/Harden-Windows-Security/releases/tag/v2023.6.29'>Hardening script update v2023.6.29</a>.</em>

**Comment by HotCakeX** [comment]
P.S The Process mitigations list can be updated independently from the hardening script.

**Comment by TESTER-sec** [comment]
Hey @HotCakeX, much gratitude and respect.

Sent you an email at your outlook address. Would you please ping me back?

**Reply by HotCakeX** [reply]
Sry for the delay, replied back!
